poly-commitment/benches/ipa.rs

```diff
@@ -8,6 +8,69 @@ use poly_commitment::{
     commitment::CommitmentCurve, ipa::SRS, utils::DensePolynomialOrEvaluations, PolyComm, SRS as _,
 };
 
+fn benchmark_msm(c: &mut Criterion) {
+    use ark_ec::{AffineRepr, VariableBaseMSM};
+    use ark_ff::PrimeField;
+
+    let mut group = c.benchmark_group("MSM");
+    let mut rng = o1_utils::tests::make_test_rng(None);
+
+    let srs = SRS::<Vesta>::create(1 << 16);
+    srs.get_lagrange_basis_from_domain_size(1 << 16);
+
+    for msm_size_log in [8, 10, 12, 14, 16].into_iter() {
```

Comment:
As for ae1034f6f5a5b84ab302becc4da8613d72a488dc, I would simply focus on sizes we care for our usecases, i.e. 16.

Reply:
I kind of want to keep it here because it's cheap, hope it's OK -- it's good to see at which value the optimisation starts to make sense, this gives a rough feeling of how big the chunk should be IMO